connectivity/FEATURE_BLE/source/cordio/source/GattServerImpl.cpp

```diff
@@ -671,6 +680,10 @@ ble_error_t GattServer::read(
         return BLE_ERROR_NONE;
     }
 
+#if TRACE_WRITE_VALUES
+    tr_debug("read %s", mbed_trace_array(buffer, buffer_length));
```

Comment:
trace should include the conn handle and the att handle
```
tr_debug("Attribute %d read on connection %d (value: %s"), att_handle, connection, mbed_trace_array(buffer, buffer_length));
```

Reply:
Good point. I'll add conn and att handles to the trace.